new ecosystem udpates

letmejustputthishere opened the pull request:
i added some of the controversial `ecocon` from the previous PR back, happy to discuss this during a call or async :)

@yizhao-ec

yizhao-ec commented:
Hey thanks for the PR!

I found a few repos where I'm not sure of their interactions with ICP, in generally they seem to be AI related and often inherit some ICP (and other crypto) plugins from eliza but I'm not sure if they interacts with ICP in a meaningful way:

* https://github.com/yarenty/kowalski
* https://github.com/suraj975/hackathon-safe-wallet
* https://github.com/julienbrs/eliza-yapper
* https://github.com/AyushBherwani1998/agentic-hackathon
* https://github.com/todoencadena/Ai-Hackathon
* https://github.com/JordanNDE/atlasAgent

Regarding the ecosystem connections:

* For `Zondax` and `Maestro` I see they both have history with non-ICP chains. Are there particular reasons why you think they are becoming more ICP aligned (outside of the few new ICP repos)?
* `inacta AG` seems not active? I do see their ICP repo although its now archived, while the remainder of their repos seems not ICP aligned and fairly old
* Why should `Galadriel` be connected to ICP? It has the plugins from eliza but seems mostly interacting with EVM/Solana.

These 2 connections looks good, but I think we may already have them already:
* Juno you're looking for is: https://crypto-ecosystems.xyz/#juno-ic 
* And `Liquidium Finance` already exists (instead of creating a new `Liquidium`) and judging by some metadata we have it they both link to the same project site at https://liquidium.fi/

letmejustputthishere commented:
> I found a few repos where I'm not sure of their interactions with ICP, in generally they seem to be AI related and often inherit some ICP (and other crypto) plugins from eliza but I'm not sure if they interacts with ICP in a meaningful way

Thanks, I removed them. It's hard to manually evaluate this. Do you have a recommendation of how to filter for this specific case as it seems to happen more often?

> For Zondax and Maestro I see they both have history with non-ICP chains. Are there particular reasons why you think they are becoming more ICP aligned (outside of the few new ICP repos)?

Zondax has contributed to the ICP protocol and ecosystem numerous times. Maestro is positioning itself as a key player in the growing BTCFi ecosystem on ICP. I think for both it's fair to say that they are strongly connected to the ICP ecosystem.

> inacta AG seems not active? I do see their ICP repo although its now archived, while the remainder of their repos seems not ICP aligned and fairly old

Is very actively contributing to a couple of major ecosystem projects. You can see references for this [here](https://www.linkedin.com/posts/crypto-oasis-middleeast_on-april-29-oisy-the-on-chain-web-wallet-activity-7328432160925732865-Mo9O?utm_source=share&utm_medium=member_desktop&rcm=ACoAACW1YhgByOZQKoJMpDXLLRrYMnnMoBQkX6E) and [here](https://www.linkedin.com/posts/inacta-ventures_join-world-computer-summit-2025-a-full-day-activity-7330173423937064960-rpTz?utm_source=share&utm_medium=member_desktop&rcm=ACoAACW1YhgByOZQKoJMpDXLLRrYMnnMoBQkX6E) for example.
 
> Why should Galadriel be connected to ICP? It has the plugins from eliza but seems mostly interacting with EVM/Solana.
Removed.

> These 2 connections looks good, but I think we may already have them already

Great catch, thanks for flagging. 


Please let me know if you have further questions / comments.
